maquetado/scss/details.scss

```diff
@@ -59,4 +51,25 @@
       border: 2px solid $sulu-green;
     }
   }
+
+  .book-cover {
+    position: relative;
+
+    .book-image {
+      height: 100%;
+      max-height: 416px;
+      max-width: 309px;
+      padding: 24px;
```

Comment:
Is it necessary this padding?

Reply:
It's not. I change that padding (conceptually wrong) and I add that padding in your container. Another padding was eliminated and add margin between brothers.